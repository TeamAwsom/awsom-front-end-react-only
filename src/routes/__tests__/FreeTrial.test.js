import React from 'react';
import FreeTrial from '../FreeTrial';
import renderer from 'react-test-renderer';

describe('FreeTrial', () => {
  it('renders as expected', () => {
    const tree = renderer.create(<FreeTrial />).toJSON();

    console.log(tree);
    expect(tree).toMatchSnapshot();
  });
});
