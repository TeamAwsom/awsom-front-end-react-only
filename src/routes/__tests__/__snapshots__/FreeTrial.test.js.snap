// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FreeTrial renders as expected 1`] = `
<div
  className="container"
>
  <div
    className="freetrial_description"
  >
    <h1>
      Let's find a great teacher in your area!
    </h1>
  </div>
  <form
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="app__form"
    >
      <div
        className="app__form_item"
      >
        <div>
          What instrument are you interested in learning? *
        </div>
        <select
          className="app__form_dropdownbox"
          name="instrument"
          onBlur={[Function]}
          onChange={[Function]}
          value=""
        >
          <option
            value=""
          >
            Select
          </option>
          <option
            value="piano"
          >
            Piano
          </option>
          <option
            value="guitar"
          >
            Guitar
          </option>
          <option
            value="voice"
          >
            Voice
          </option>
          <option
            value="woodwinds"
          >
            Woodwinds
          </option>
          <option
            value="brass"
          >
            Brass
          </option>
          <option
            value="drums"
          >
            Drums
          </option>
          <option
            value="strings"
          >
            Strings
          </option>
          <option
            value="chamber"
          >
            Chamber
          </option>
        </select>
      </div>
      <div
        className="app__form_item"
      >
        <div>
          Which musical style are you most interested in learning? *
        </div>
        <select
          className="app__form_dropdownbox"
          name="musicStyle"
          onBlur={[Function]}
          onChange={[Function]}
          value=""
        >
          <option
            value=""
          >
            Select
          </option>
          <option
            value="classical"
          >
            Classical
          </option>
          <option
            value="jazz"
          >
            Jazz
          </option>
          <option
            value="rock"
          >
            Rock
          </option>
          <option
            value="opera"
          >
            Opera
          </option>
        </select>
      </div>
      <div
        className="app__form_item"
      >
        <div>
          Previous musical experience: *
        </div>
        <select
          className="app__form_dropdownbox"
          name="experienceLevel"
          onBlur={[Function]}
          onChange={[Function]}
          value=""
        >
          <option
            value=""
          >
            Select
          </option>
          <option
            value="beginner"
          >
            Beginner: 0-2 years
          </option>
          <option
            value="intermediate"
          >
            Intermediate: 3-4 years
          </option>
          <option
            value="advanced"
          >
            Advanced: 5+ years
          </option>
        </select>
      </div>
      <div
        className="app__form_item"
      >
        <div>
          Do you have an instrument already? *
        </div>
        <select
          className="app__form_dropdownbox"
          name="hasInstrument"
          onBlur={[Function]}
          onChange={[Function]}
          value=""
        >
          <option
            value=""
          >
            Select
          </option>
          <option
            value={true}
          >
            yes
          </option>
          <option
            value={false}
          >
            no
          </option>
        </select>
      </div>
      <div
        className="app__form_item"
      >
        <div>
          Address 1 *
        </div>
        <input
          className="app__form_input"
          name="addressOne"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="app__form_item"
      >
        <div>
          Address 2
        </div>
        <input
          className="app__form_input"
          name="addressTwo"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="app__form_item"
      >
        <div>
          City *
        </div>
        <input
          className="app__form_input"
          name="city"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="app__form_item"
      >
        <div>
          State *
        </div>
        <input
          className="app__form_input"
          name="state"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="app__form_item"
      >
        <div>
          Zip *
        </div>
        <input
          className="app__form_input"
          name="zip"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="app__form_item"
      >
        <div>
          Availability *
        </div>
        <div>
          <select
            className="app__form_dropdownbox"
            name="availability[0].day"
            onBlur={[Function]}
            onChange={[Function]}
            value=""
          >
            <option
              value=""
            >
              Select
            </option>
            <option
              value="monday"
            >
              Monday
            </option>
            <option
              value="tuesday"
            >
              Tuesday
            </option>
            <option
              value="wednesday"
            >
              Wednesday
            </option>
            <option
              value="thursday"
            >
              Thursday
            </option>
            <option
              value="friday"
            >
              Friday
            </option>
            <option
              value="saturday"
            >
              Saturday
            </option>
            <option
              value="sunday"
            >
              Sunday
            </option>
          </select>
          <span />
          <select
            name="availability[0].fromTime"
            onBlur={[Function]}
            onChange={[Function]}
            value={0}
          >
            <option
              value=""
            >
              From
            </option>
            <option
              value={9}
            >
              9:00 AM
            </option>
            <option
              value={10}
            >
              10:00 AM
            </option>
            <option
              value={11}
            >
              11:00 AM
            </option>
            <option
              value={12}
            >
              12:00 PM
            </option>
            <option
              value={13}
            >
              1:00 PM
            </option>
            <option
              value={14}
            >
              2:00 PM
            </option>
            <option
              value={15}
            >
              3:00 PM
            </option>
            <option
              value={16}
            >
              4:00 PM
            </option>
            <option
              value={17}
            >
              5:00 PM
            </option>
            <option
              value={18}
            >
              6:00 PM
            </option>
            <option
              value={19}
            >
              7:00 PM
            </option>
            <option
              value={20}
            >
              8:00 PM
            </option>
            <option
              value={21}
            >
              9:00 PM
            </option>
          </select>
          <span />
          <select
            name="availability[0].toTime"
            onBlur={[Function]}
            onChange={[Function]}
            value={0}
          >
            <option
              value=""
            >
              To
            </option>
            <option
              value={9}
            >
              9:00 AM
            </option>
            <option
              value={10}
            >
              10:00 AM
            </option>
            <option
              value={11}
            >
              11:00 AM
            </option>
            <option
              value={12}
            >
              12:00 PM
            </option>
            <option
              value={13}
            >
              1:00 PM
            </option>
            <option
              value={14}
            >
              2:00 PM
            </option>
            <option
              value={15}
            >
              3:00 PM
            </option>
            <option
              value={16}
            >
              4:00 PM
            </option>
            <option
              value={17}
            >
              5:00 PM
            </option>
            <option
              value={18}
            >
              6:00 PM
            </option>
            <option
              value={19}
            >
              7:00 PM
            </option>
            <option
              value={20}
            >
              8:00 PM
            </option>
            <option
              value={21}
            >
              9:00 PM
            </option>
          </select>
          <button
            className="btn"
            onClick={[Function]}
            type="button"
          >
            <i
              className="far fa-times-circle"
            />
          </button>
        </div>
        <div
          className="app__form_container"
        >
          <button
            className="btn btn-secondary"
            onClick={[Function]}
            type="button"
          >
            Add Time
          </button>
        </div>
      </div>
      <div
        className="app__form_item"
      >
        <div>
          Allergies:
        </div>
        <input
          className="app__form_input"
          name="allergies"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="app__form_item"
      >
        <div>
          Special Needs:
        </div>
        <input
          className="app__form_input"
          name="specialNeeds"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="app__form_container"
      >
        <button
          className="btn btn-primary "
          disabled={false}
          type="submit"
        >
          SUBMIT
        </button>
      </div>
    </div>
  </form>
</div>
`;
